sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1058,7 +1058,11 @@ private void updateLatestOffsets() {
           long offset = offsetConsumer.position(p.topicPartition);
           p.setLatestOffset(offset);
         } catch (Exception e) {
-          LOG.warn("{}: exception while fetching latest offsets. ignored.",  this, e);
+          if (closed.get()) {
+            break;
```

Comment:
np, according to the Beam contribution guide committers can do such things sometimes, and I thought it wasn't worth waiting for more ping-pong on comments 😉 (especially since I'm in an opposite timezone).